### ISSUE TITLE
Add support to duckdb

### ISSUE THREAD

**Issue by jpmarindiaz** [dgrtwo/dbcooper#10]
Hi
I am trying to use `dbcooper` with a duckdb database. When querying a table with `mydb_mytable()` I get the following error:

```
Error: Parser Error: syntax error at or near "SELECT"
LINE 2: FROM SELECT * FROM  region AS q01
             ^
```
Note that I am able to run `tbl(con, "mytable")` without problem
Any ideas?

**Comment by chriscardillo** [comment]
Hi @jpmarindiaz - ~Can you send through how you are instantiating?~ Looks to be an issue with the `dbc_table` function.

~Can you try installing the duckdb branch and let me know what you think?~

This appears to be an upstream issue and I've logged it in the `dbplyr` repo: https://github.com/tidyverse/dbplyr/issues/1173

**Comment by chriscardillo** [comment]
@dgrtwo - ~I think this [dbc_table logic](https://github.com/chriscardillo/dbcooper/blob/fb380379a8582c8692c9eb99a873dd3d662d3ff6/R/table_functions.R#L15-L29) could be simplified to just `return(dplyr::tbl(con, table_name))`. Confirming this works on the [duckdb branch](https://github.com/chriscardillo/dbcooper/blob/9d25bb1d9c515d47b5d272d787099ea69cd3c7b5/R/table_functions.R#L8), if you want to take a look.~

~Wanted to ask, what motivated this? Is there an edge case we would regress on if we simplified the `dbc_table` logic?~

EDIT: This simpler logic doesn't work with Snowflake. I think this is actually a dplyr/dbplyr issue, and I opened a ticket.

**Comment by chriscardillo** [comment]
@jpmarindiaz - Thanks for your patience. The duckdb branch has fixes in place. I will merge to main soon.

@dgrtwo - For your review! https://github.com/chriscardillo/dbcooper/pull/14

**Comment by chriscardillo** [comment]
Fixed by #14
